### DISCUSSION TITLE
How to get time-series from the Snapshot API?

### DISCUSSION THREAD

**Discussion by amanwithwings** [snapshot-labs/snapshot#3328] [Q&A] [answered]
Hey, 

Let me begin by saying that I'm a noob when it comes to programming. I'm using this interface to get some data about Snapshot spaces: https://hub.snapshot.org/graphql. I need: 
1. Number of Snapshot spaces- I was able to narrow down on this using _skip_
2. Growth of Snapshot spaces, in number of new spaces (month on month) - did not get this
3. Active Snapshot spaces, ( I was thinking of defining a limit of atleast 1 proposal every month)- did not get this

I'd be grateful for any help on getting this info :)

**Comment by metrox-eth** [comment, marked as answer]
You may want to have a look at deepdao.io
They've done an excellent job at gathering DAOs data
https://twitter.com/DeepDAO_io/status/1595728165468786688
